src/astar.rs

```diff
@@ -71,63 +71,66 @@ pub fn astar<G, F, H, K, IsGoal>(
     mut estimate_cost: H,
 ) -> Option<(K, Vec<G::NodeId>)>
 where
-    G: IntoEdges + Visitable,
+    G: IntoEdges,
```

Comment:
```suggestion
    G: IntoEdges + Visitable,
```

I would just keep this in the interface for flexibility, even if it's no longer used.